node/src/accountant/payment_adjuster/non_unit_tests/mod.rs

```diff
@@ -441,16 +500,16 @@ fn make_agent(cw_service_fee_balance: u128) -> BlockchainAgentMock {
         .service_fee_balance_minor_result(cw_service_fee_balance)
         // For PaymentAdjuster itself
         .service_fee_balance_minor_result(cw_service_fee_balance)
-        .agreed_transaction_fee_margin_result(Percentage::new(15))
+        .gas_price_margin_result(TX_FEE_MARGIN_IN_PERCENT.clone())
```

Comment:
You should try eliminating clone. Idea from your side: to implement `Copy` and then use `*` here.